src/test/java/junit/CreateNewOpportinities.java

```diff
@@ -0,0 +1,57 @@
+package test.junit;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import pages.*;
+
+/**
+ * Created by Fernando Nakamura on 6/26/2015.
+ */
+public class CreateNewOpportunities {
+
+    private LoginPage loginPage;
+    private MainApp mainApp;
+
+    private AppBody appOpportunities;
+
+    @Before
+    public void setUp() {
+        loginPage = new LoginPage();
+        String email = "f_nak@hotmail.com";
+        String password = "abc123456";
+        String userName = "Fernando Nakamura";
+
+        mainApp = loginPage.loginAs(email,password,userName);
```

Comment:
Consider using a method loginAsPrimaryUser() in order to avoid declaring credential hard coded in the test cases